iceoryx_posh/source/popo/ports/client_port_data.cpp

```diff
@@ -35,7 +37,8 @@ ClientPortData::ClientPortData(const capro::ServiceDescription& serviceDescripti
     : BasePortData(serviceDescription, runtimeName, clientOptions.nodeName)
     , m_chunkSenderData(memoryManager, clientOptions.serverTooSlowPolicy, HISTORY_CAPACITY_ZERO, memoryInfo)
     , m_chunkReceiverData(getResponseQueueType(clientOptions.responseQueueFullPolicy),
-                          clientOptions.responseQueueFullPolicy)
+                          clientOptions.responseQueueFullPolicy,
+                          memoryInfo)
```

Comment:
memoryInfo is still used for nothing, or?

Reply:
correct